ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -64,15 +79,81 @@ def set_waypoint_velocity(self, waypoints, waypoint, velocity):
 
     def distance(self, waypoints, wp1, wp2):
         dist = 0
-        dl = lambda a, b: math.sqrt((a.x-b.x)**2 + (a.y-b.y)**2  + (a.z-b.z)**2)
         for i in range(wp1, wp2+1):
-            dist += dl(waypoints[wp1].pose.pose.position, waypoints[i].pose.pose.position)
+            dist += euclidean_dist(waypoints[wp1].pose.pose.position, waypoints[(i)].pose.pose.position)
             wp1 = i
         return dist
 
+    def euclidean_dist(self, pt1, pt2):
+        """
```

Comment:
Great initiative to comment that method, I'll would personally like to have one for each classes & functions